client/my-sites/email/email-management/email-home.tsx

```diff
@@ -0,0 +1,206 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import { useSelector } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import EmptyContent from 'calypso/components/empty-content';
+import Main from 'calypso/components/main';
+import SectionHeader from 'calypso/components/section-header';
+import { useGetEmailDomainsQuery } from 'calypso/data/emails/use-get-email-domains-query';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { hasGSuiteWithUs } from 'calypso/lib/gsuite';
+import { hasTitanMailWithUs } from 'calypso/lib/titan';
+import EmailHeader from 'calypso/my-sites/email/email-header';
+import EmailListActive from 'calypso/my-sites/email/email-management/home/email-list-active';
+import EmailListInactive from 'calypso/my-sites/email/email-management/home/email-list-inactive';
+import EmailNoDomain from 'calypso/my-sites/email/email-management/home/email-no-domain';
+import EmailPlan from 'calypso/my-sites/email/email-management/home/email-plan';
+import EmailProvidersComparisonStacked from 'calypso/my-sites/email/email-providers-stacked-comparison';
+import { emailManagementTitanSetUpMailbox } from 'calypso/my-sites/email/paths';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import hasLoadedSites from 'calypso/state/selectors/has-loaded-sites';
+import { createSiteDomainObject } from 'calypso/state/sites/domains/assembler';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { ResponseDomain } from 'calypso/lib/domains/types';
+import type { TranslateResult } from 'i18n-calypso';
+import type { ReactElement } from 'react';
+
+import './style.scss';
+
+interface ContentWithHeaderProps {
+	children: ReactElement;
+}
+
+const ContentWithHeader = ( { children }: ContentWithHeaderProps ): ReactElement => {
```

Comment:
I don't mind leaving it like this at all but given that this type is not used anywhere else and it is relatively short I would just inline it:

```suggestion
const ContentWithHeader = ( { children: ReactElement } ): ReactElement => {
```